File: php/problem1.php
<?php
	$sum = 0;
	$i = 0;
	
	while($i<1000)
	{
		if(($i%3==0) or ($i%5==0))
		{
			$sum = $sum+$i;
		}
		$i++;
	}
	echo $sum;
?>

File: java/problem1.java
class Problem1{
	public static void main(String args[]){
		int sum=0;
		for(int i=0; i<=999;i=i+1){
			if( i % 3 == 0 || i % 5 == 0){
				sum = sum+i;
			}
		}
		System.out.println(sum);
	}
}
